components/Navigation/Navigation.tsx

```diff
@@ -71,7 +109,10 @@ export const NavigationDrawer = styled('nav', {
   flexDirection: 'column',
 
   variants: {
-    elevation: elevationVariant,
+    elevation: Object.keys(elevationVariant).map((_, index) => ({
+      ...elevationVariant[index],
+      bc: `$${index < 10 ? '0' : ''}${index}dp`,
```

Comment:
could be refactored into a helper function, I expect it might be used elsewhere soon

Reply:
I would go on factorizing code when it will be needed instead. Depending on usages factorization would change.